controllers/start.go

```diff
@@ -28,12 +33,24 @@ func StartManager(cfg ControllerConfig) {
 		atom.SetLevel(zap.DebugLevel)
 	}
 
-	logger := ctrlzap.New(func(opts *ctrlzap.Options) {
+	zapLogger := ctrlzap.NewRaw(func(opts *ctrlzap.Options) {
 		opts.Level = &atom
 		opts.Development = true
 	})
+	logger := zapr.NewLogger(zapLogger)
 	ctrl.SetLogger(logger)
 
+	newrelicApp, err := newrelic.NewApplication(
```

Comment:
Shouldn't this be optional? If I try to debug locally, then won't this fail?

Reply:
The first option `ConfigEnabled(false)` sets the default to disabled and the last uses environment variables (`NEW_RELIC_ENABLED`) for a toggle. Ideally it shouldn't matter at all locally, but let me know if it does.